test/code/plugins/squid_log_parser_plugintest.rb

```diff
@@ -0,0 +1,150 @@
+## squid_log_parser_plugintest
+## Created by Alessandro Cardoso
+## 
+
+
+#require 'fluent'
+#require 'fluent/parser'
+require 'test/unit'
+require_relative '../../../source/code/plugins/squidlogparser.rb'
+
+class SquidLogParserTestRuntimeError < SquidLogParser      #Fluent::SquidLogParser
+  def log_error(text)
```

Comment:
If you look at the auditlog_lib_test.rb it does a similar call as I am doing :  @@squidlog = Fluent::SquidLogParser.new(SquidLogParserTestRuntimeError.new)